index.js

```diff
@@ -61,3 +61,5 @@ export { default as NoteCreatePage } from './lib/Notes/NoteCreatePage';
 export { default as NoteViewPage } from './lib/Notes/NoteViewPage';
 export { default as NoteEditPage } from './lib/Notes/NoteEditPage';
 export { default as NotesSmartAccordion } from './lib/Notes/NotesSmartAccordion';
+
+export * from './lib/PluginFindRecord';
```

Comment:
Absolute nit-picking, but could we export explicitly from here? It's useful, when checking what things are called and what gets exported to just look at the top-level `index.js` and not have to dive into subdirectories etc.